### ISSUE TITLE
Arkmanager update not detecting available update many hours after it is posted.

### ISSUE THREAD

**Issue by MatthewMedic** [FezVrasta/ark-server-tools#142]
I'm having trouble with the arkmanager update command, even with force.

SteamDB shows that ark dedicated server's public depot was updated 5 hours ago with build 721645, but when I do an arkmanager update it says my server is up to date with build 712950.

This means my players who are playing through steam get the update and then cannot log in because I can't update the server unless I do it manually. (which appears to break arkmanager start/stop/status commands and possibly others I haven't tried)

**Comment by klightspeed** [comment]
By "manually updating", do you mean:
```
./steamcmd.sh +login anonymous +force_install_dir "$HOME/ARK" +app_update 376030 +quit
```

The only way manually updating the server would break arkmanager start/stop/status would be if the ARK server directory were moved or removed.  start/stop would break if `~/ARK/ShooterGame/Binaries/Linux/ShooterGameServer` were missing, and status would break if `~/ARK/steamapps/appmanifest_376030.acf` were missing.

As for the failure to update, which of the following exists:
* ~/Steam/appcache/appinfo.vdf
* ~/steamcmd/appcache/appinfo.vdf
* ~/.local/share/Steam/appcache/appinfo.vdf

**Comment by MatthewMedic** [comment]
Yup that's exactly what I mean by updating manually. ShooterGameServer was not missing, in fact every file was binary identical. The only difference was permissions. 

~/Steam/appcache has a packageinfo.vdf but no appinfo.vdf
There is no ~/steamcmd/appcache directory on my system.
There is also no ~/.local/share/Steam.

**Comment by Saiboogu** [comment]
>The only difference was permissions.

That's a significant difference. Manually update, adjust perms back to match what arkmanager had originally? As a workaround.

**Comment by klightspeed** [comment]
@MatthewMedic did you run the update under the user that you are running arkmanager under, under another user, or as root?

By changed permissions, did the file owners / groups change, or did their mode change - e.g. if you're running arkmanager under the user `steam`, does `stat -c '%A %U %G' /home/steam/ARK` give the following or something different:
```
drwxr-xr-x steam steam
```

**Comment by MatthewMedic** [comment]
@klightspeed I did run it under the same user yes. I never run things as root unless absolutely necessary.

The file mode changed on some files, I can't run that because after that incident I backed up the server data, wiped the install and installed it again with arkmanager.

Thinking back on it now it's possible the manual update removed execution permission on ShooterGameServer somehow and I just didn't look closely enough. I was in a bit of a rush trying to get the server updated and online again. 

When I have a moment I'll try doing what I did again and see what the difference was.

Until then I'll close this issue because I'm not being very helpful at detailing it and the manual update breaking isn't really related to arkmanager.

I really just wanted to point out the fact that update and update with -force were not updating when an update was indeed available.
